Bottom: 95e5672b22649506129764ac6624328970ee6a54
Top:    1e615992ff9ab98d8784cddbfb22f6467a63a227
Author: Lawrence D'Anna <lawrence_danna@apple.com>
Date:   2019-10-03 12:32:18 -0700

Refresh of allow-arbitrary-python-streams

---
diff --git a/lldb/source/Plugins/ScriptInterpreter/Python/PythonDataObjects.cpp b/lldb/source/Plugins/ScriptInterpreter/Python/PythonDataObjects.cpp
index 643e4612118..005bac5a0cf 100644
--- a/lldb/source/Plugins/ScriptInterpreter/Python/PythonDataObjects.cpp
+++ b/lldb/source/Plugins/ScriptInterpreter/Python/PythonDataObjects.cpp
@@ -1232,7 +1232,7 @@ namespace {
 class PythonIOFile : public OwnedPythonFile<PresumptivelyValidFile> {
 public:
   PythonIOFile(const PythonFile &file, bool borrowed)
-      : OwnedPythonFile(file, borrowed){};
+      : OwnedPythonFile(file, borrowed) {}
 
   ~PythonIOFile() override { Close(); }
 
@@ -1245,7 +1245,7 @@ public:
     if (PyErr_Occurred())
       return Status(llvm::make_error<PythonException>("Close"));
     return Status();
-  };
+  }
 
   Status Flush() override {
     GIL takeGIL;
